packages/payment-processor/src/payment/utils.ts

```diff
@@ -345,3 +346,27 @@ export function comparePnTypeAndVersion(
     pn?.version === getPaymentNetworkExtension(request)?.version
   );
 }
+
+/**
+ * set approval to 0 of a token from a user (as wallet address provider)
+ */
+export async function resetApproveErc20Utils(
```

Comment:
Would be best to rename the function `revokeErc20Approval`.

`revoke` is the term used in MM when setting an approval to 0.
`Erc20Approval` is more natural.
`Utils` adds complexity to the name, IMO.